os/net/netmgr/netdev_wifi.c

```diff
@@ -145,6 +145,20 @@ int _trwifi_handle_command(struct netdev *dev, lwnl_req cmd)
 	return 0;
 }
 
+/* it's for reloading operation in binary manager.
+ * it must be used to binary manager only.
+ */
+trwifi_result_e netdev_deinit_wifi(void)
```

Comment:
Is it okay not to define this in header file?

Reply:
It's function only for binary manager. So it's better not to define it to a header file.